src/main/java/com/alkemy/ong/model/request/NewsRequest.java

```diff
@@ -0,0 +1,19 @@
+package com.alkemy.ong.model.request;
+
+import javax.validation.constraints.NotBlank;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class NewsRequest {
+
+  @NotBlank(message = "The field name cannot be blank.")
```

Comment:
Para ser consistente con los mensajes que existen en el proyecto lo modificaría a: "Name cannot be null or empty."